src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -630,6 +630,12 @@
     <None Include="ConverterTestData\StaticDriverVerifier\cancelspinlock_bug1.tt.sarif">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="ConverterTestData\PyLint\PyLint-01.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Where did you get this file? I just want to make sure that it does not contain diagnostics for Microsoft proprietary code.

Reply:
@komal436 Sorry, I wasn't clear. *Which* Python file did you run Pylint on? Was it Microsoft proprietary code?

Reply:
@lgolding  I ran Pylint on the local python local that I created.